_datasets/json/global-fao-asi-2025-11-04.json

```diff
@@ -0,0 +1,145 @@
+          }
+        },
+        {
+          "id": "attribution_contact_point_1759330864916",
+          "role": "contact_point",
+          "entity": {
+            "name": "Mattia Amadio",
+            "email": "mamadio@worldbank.org",
+            "url": ""
+          }
+        },
+        {
+          "id": "attribution_1762252073805",
+          "role": "resource_provider",
+          "entity": {
+            "name": "Mattia Amadio",
+            "email": "mamadio@worldbank.org",
+            "url": ""
+          }
+        }
+      ],
+      "sources": [],
+      "referenced_by": [],
+      "resources": [
+        {
+          "title": "Drought frequency",
+          "description": "Aggregation of annual data (1984-2024) into frequency-intensity summary layers for the main crop season (S1) and secondary crop season (S2). Two indicators are provided: \n-    30 percent (1/3) of cropland area being affected by the drought event\n -   50 percent (1/2) of cropland area being affected by the drought event",
+          "media_type": "",
+          "format": "geotiff",
+          "spatial_resolution": "1000",
+          "coordinate_system": "EPSG:4326",
+          "access_url": "https://www.fao.org/giews/earthobservation/",
+          "download_url": "https://datacatalogfiles.worldbank.org/ddh-published/0066907/DR0095589/ASI.zip",
+          "temporal": {
+            "start": "1984",
+            "end": "2024",
+            "duration": "P40Y",
+            "temporal_resolution": "P1Y"
+          },
+          "id": "resource_LwLJRHNJ"
+        }
+      ],
+      "hazard": {
+        "event_sets": [
+          {
+            "hazards": [
+              {
+                "type": "drought",
+                "processes": "agricultural_drought",
+                "intensity_measure": "VCI:-",
+                "trigger": "",
+                "id": "hazard_Doc2hzTg"
+              }
+            ],
+            "events": [],
+            "type": "drought",
+            "processes": "agricultural_drought",
+            "intensity_measure": "VCI:-",
+            "trigger": {
+              "type": "",
+              "processes": ""
+            },
+            "analysis_type": "empirical",
+            "frequency_distribution": "",
+            "seasonality": "uniform",
+            "calculation_method": "inferred",
+            "event_count": "",
+            "occurrence_range": "",
+            "temporal": {
+              "start": "",
+              "end": "",
+              "duration": "",
+              "temporal_resolution": ""
+            },
+            "id": "event_set_uPMH0sXy"
+          }
+        ]
+      },
+      "links": [],
+      "publisher": {
+        "name": "",
+        "email": "",
+        "url": ""
+      },
+      "contact_point": {
+        "name": "",
+        "email": "",
+        "url": ""
+      },
+      "creator": {
+        "name": "",
+        "email": "",
+        "url": ""
+      }
```

Comment:
@matamadio This isn't causing any issues, but I think the editor shouldn't produce the creator, publisher, and contact_point objects since the schema only has them under the attributions array, right? This is just a thought, but it might make things cleaner for people looking at these JSON in the future if the editor validates that there aren't any extra keys in the metadata that don't belong on the schema.